README.rst

```diff
@@ -1,23 +1,182 @@
+                 [--certificates-collection CERTIFICATES_COLLECTION]
+                 [--gfx-bucket GFX_BUCKET] [--gfx-collection GFX_COLLECTION]
+                 [--addons-bucket ADDONS_BUCKET]
+                 [--addons-collection ADDONS_COLLECTION]
+                 [--plugins-bucket PLUGINS_BUCKET]
+                 [--plugins-collection PLUGINS_COLLECTION] [--app APP]
+                 [-o OUT]
+
+Build a blocklists.xml file from Kinto blocklists.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -s SERVER, --server SERVER
+                        The location of the remote server (with prefix)
+  -a AUTH, --auth AUTH  BasicAuth token:my-secret
+  -v, --verbose         Show all messages.
+  -q, --quiet           Show only critical errors.
+  -D, --debug           Show all messages, including debug messages.
+  --certificates-bucket CERTIFICATES_BUCKET
+                        Bucket name for certificates
+  --certificates-collection CERTIFICATES_COLLECTION
+                        Collection name for certificates
+  --gfx-bucket GFX_BUCKET
+                        Bucket name for gfx
+  --gfx-collection GFX_COLLECTION
+                        Collection name for gfx
+  --addons-bucket ADDONS_BUCKET
+                        Bucket name for addons
+  --addons-collection ADDONS_COLLECTION
+                        Collection name for addon
+  --plugins-bucket PLUGINS_BUCKET
+                        Bucket name for plugins
+  --plugins-collection PLUGINS_COLLECTION
+                        Collection name for plugin
+  --app APP             Targeted blocklists.xml APP id
+  -o OUT, --out OUT     Output XML file.
+
+
+Show differences between two XML files
+======================================
+
+Once you've imported the data and exported them, you may want to
+validate that both Kinto and addons server export the exact same data in XML.
+
+You can use the ``xml-verifier`` command to validate that both files
+are rendered the same way.
+
+The **xml-verifier** command accept both files path and files URLs.
 
-Generate a blocklist.xml file from Kinto collections.
 
 .. code-block::
 
-    usage: kinto2xml [-h] [-s SERVER] [-a AUTH] [-b BUCKET] [-v] [-q] [-D]
-                     [-o OUT]
-
-    Build a blocklists.xml file.
-
-    optional arguments:
-      -h, --help            show this help message and exit
-      -s SERVER, --server SERVER
-                            The location of the remote server (with prefix)
-      -a AUTH, --auth AUTH  BasicAuth token:my-secret
-      -b BUCKET, --bucket BUCKET
-                            Bucket name.
-      -v, --verbose         Show all messages.
-      -q, --quiet           Show only critical errors.
-      -D, --debug           Show all messages, including debug messages.
-      -o OUT, --out OUT     Output file.
+    APPID="{ec8030f7-c20a-464f-9b0e-13a3a9e97384}"
+    kinto2xml --app $APPID -o generated-blocklists.xml
```

Comment:
It depends of you setup, but default should work. (json2kinto and kinto2xml uses the same defaults.)